benches/benches.rs

```diff
@@ -0,0 +1,15 @@
+#![allow(dead_code)] // Due to criterion we need this to avoid warnings.
+
+#[macro_use]
+extern crate criterion;
+extern crate raft;
+
+mod suites;
+use suites::*;
+
+criterion_main!(
```

Comment:
the default sample size for the criterion is 100, maybe we can use a smaller sample size like 10 to reduce the benchmark time.